src/main/java/com/google/sps/servlets/FindLocationServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/find-location")
```

Comment:
include descriptions for each of these .. when will they be requested? What params will they need to accept?

Reply:
Done!